tests/common.py

```diff
@@ -31,25 +31,23 @@ def make_barebones_config() -> dict:
                 "share_branch": False,
             },
         },
+        "science_configurations": {
+            "sci0": {
+                "cable": {
```

Comment:
Be careful! This breaks `adjust_namelist_file` for now since it expects the configs without the `cable` level. But leave it in since that's coming with the next PR.